department-of-veterans-affairs/va.gov-cms
# VACMS-2312: User guides content model part 2, and theming.

## Description

See #2312 and #2438 

## Testing done
Manual


![User_guide_strategy_•_Agile_Six_Workspace](https://user-images.githubusercontent.com/643678/88617895-b2e75200-d065-11ea-959a-185220fbcbf9.jpg)

## QA steps

As admin 

1. Edit /node/6501/edit, a Detail page
1. Add a content block
  - [x] In the paragraphs browser, you SHOULD NOT see "Lists of links"
1. Edit /node/6454/edit, a CMS user guide page
1. Add content block
  - [x] In the paragraphs browser note the new content block type "Lists of links"
 a section header for each list of links
  - [x] Add a "list of link teasers" and 6 link teasers. Add an optional section header. 
  - [x] Add a "Lists of links" and add 4 lists of links
     - [ ] Add a "final link" for 2 of these 4.
     - [ ] Add an optional section header
1. Add two "Related user guides"

1. Save


Review ACs from #2312 
  - [x] Add H1 to top of proofing block (done previously)
  - [x] Remove irrelevant blocks: Recent changes, Preview button, Owner (done previously)
  - [x] Remove broken links report from documentation content type (done previously)
  - [x] Fix breadcrumb bug (breadcrumbs not showing past first level) (done previously)
  - [ ] Adjust font size for headings (headings are now smaller)
  - [ ] Implement lite CSS adjustments(line spacing btwn headings, padding btwn text and tables)
  - [ ] Limit body column width to ~95 characters for readability

New ACs based on #2438 content model updates
- [ ] List of links should have an h3 section header
